src/features/auth/signup/SetBirthday/DatePicker/DatePickerSpinner.web.test.tsx

```diff
@@ -0,0 +1,67 @@
+import mockdate from 'mockdate'
+import React from 'react'
+
+import { formatDateToISOStringWithoutTime } from 'libs/parsers'
+import { fireEvent, render } from 'tests/utils/web'
+
+import { DatePickerSpinner } from './DatePickerSpinner'
+
+const CURRENT_DATE = new Date('2020-12-01T00:00:00.000Z')
+const ELIGIBLE_AGE_DATE = new Date('2003-12-01T00:00:00.000Z')
+
+const props = { goToNextStep: jest.fn(), signUp: jest.fn() }
+
+jest.mock('features/auth/settings')
+
+// eslint-disable-next-line local-rules/no-allow-console
+allowConsole({ warn: true })
```

Comment:
tu peux t'en passer ou pas du allowConsole ? t'as des warnings dans les tests ?

Reply:
Si je m'en passe, les tests fails et j'ai ces messages. Je pense que c'est lié à la lib `react-mobile-picker`
```
  console.warn
    Warning: "year" doesn't contain an option of "2020".

      at console.methodSpy (node_modules/console-fail-test/src/spies/jest.js:17:35)
      at t.computeTranslate (node_modules/react-mobile-picker/lib/react-mobile-picker.js:1:3295)
      at new t (node_modules/react-mobile-picker/lib/react-mobile-picker.js:1:1891)
      at constructClassInstance (node_modules/react-dom/cjs/react-dom.development.js:12716:18)

  console.warn
    Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://reactjs.org/link/unsafe-component-lifecycles for details.
    
    * Move data fetching code or side effects to componentDidUpdate.
    * If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://reactjs.org/link/derived-state
    * Rename componentWillReceiveProps to UNSAFE_componentWillReceiveProps to suppress this warning in non-strict mode. In React 18.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.
    
    Please update the following components: t

      at console.methodSpy (node_modules/console-fail-test/src/spies/jest.js:17:35)
      at printWarning (node_modules/react-dom/cjs/react-dom.development.js:67:30)
      at warn (node_modules/react-dom/cjs/react-dom.development.js:34:5)
      at Object.<anonymous>.ReactStrictModeWarnings.flushPendingUnsafeLifecycleWarnings (node_modules/react-dom/cjs/react-dom.development.js:11530:7)

  console.warn
    Warning: "year" doesn't contain an option of "2020".

      at console.methodSpy (node_modules/console-fail-test/src/spies/jest.js:17:35)
      at t.computeTranslate (node_modules/react-mobile-picker/lib/react-mobile-picker.js:1:3295)
      at new t (node_modules/react-mobile-picker/lib/react-mobile-picker.js:1:1891)
      at constructClassInstance (node_modules/react-dom/cjs/react-dom.development.js:12716:18)
```

Reply:
`Warning: "year" doesn't contain an option of "2020".` a l'air assez facile à corriger non ? essaye 5 min et si t'arrives pas tant pis.
Okey pour l'autre

Reply:
Je ne vois pas comment corriger `Warning: "year" doesn't contain an option of "2020".` :/